pandas/core/internals/blocks.py

```diff
@@ -2179,15 +2179,33 @@ def _na_value(self):
     def fill_value(self):
         return tslibs.iNaT
 
+    def to_dense(self):
+        # TODO(DatetimeBlock): remove
+        return np.asarray(self.values)
+
     def get_values(self, dtype=None):
         """
         return object dtype as boxed values, such as Timestamps/Timedelta
         """
         if is_object_dtype(dtype):
-            return lib.map_infer(self.values.ravel(),
-                                 self._box_func).reshape(self.values.shape)
+            values = self.values
+
+            if self.ndim > 1:
+                values = values.ravel()
+
+            values = lib.map_infer(values, self._box_func)
+
+            if self.ndim > 1:
```

Comment:
isn't this reshape *always* true?

Reply:
I don't know.  The gymnastics in internals to get around the fact that we can't reshape to (nrows, 1) is over my head.

Reply:
ok, it *was* ok before, puzzled why it needs changing now

Reply:
For Series `ndim` will be 1

```python
In [2]: ser = pd.Series(pd.date_range('2000', periods=4, tz="UTC"))

In [3]: ser._data.blocks[0].ndim
Out[3]: 1
```

Or perhaps I misunderstood the question.